docparse/find.go

```diff
@@ -340,6 +340,17 @@ func GetReference(prog *Program, context string, isEmbed bool, lookup, filePath
 			arLookup = fmt.Sprintf("[%v:%v]", wrapper, arLookup)
 		}
 		return GetReference(prog, context, isEmbed, arLookup, filePath)
+	case *ast.MapType:
+		// TODO, create the additional properties thing
+		var mp *ast.MapType = typ
+		ident, ok := mp.Value.(*ast.Ident)
+		if ok {
+			fmt.Printf("# ***** Is Ident %#v\n", ident)
+			panic("Is Ident")
```

Comment:
Some cleanup needed?

- remove fmt.Printf
- return errors instead of panic

Reply:
Indeed. Will clean that up (it looks like in my tests I did not have that kind of maps).